<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page_inicio">
     <widget class="QLabel" name="titulo_label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DUNE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_confirmar">
      <property name="geometry">
       <rect>
        <x>488</x>
        <y>414</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>351</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="layout_grid">
       <item row="0" column="0">
        <widget class="QLabel" name="IP_label">
         <property name="text">
          <string>IP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="IP_text_input"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="puerto_label">
         <property name="text">
          <string>Puerto a conectar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="puerto_text_input"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="nombre_label">
         <property name="text">
          <string>Nombre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="nombre_text_input"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_crear_unirse">
     <widget class="QPushButton" name="button_crear_partida">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear Partida</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_unirse_partida">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Unirse a Partida</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_crear">
     <widget class="QPushButton" name="button_cofirmar_cant_nombre">
      <property name="geometry">
       <rect>
        <x>538</x>
        <y>300</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>471</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="nombre_partida_label">
         <property name="text">
          <string>Ingresar nombre de la partida</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nombre_partida_input"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cantidad_jugadores_label">
         <property name="text">
          <string>Ingresar cantidad de jugadores</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="cantidad_jugadores_input"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_lista_patidas">
     <widget class="QListWidget" name="lista_partidas">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_casas">
     <widget class="QLabel" name="bienvenida_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>BIenvenido a Dune nombre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select your house</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_volver">
      <property name="geometry">
       <rect>
        <x>38</x>
        <y>414</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>461</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="atreides_pic_label">
         <property name="text">
          <string>Pic_Atreides</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="ordos_pic_label">
         <property name="text">
          <string>Pic_Ordos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="harkonnen_pic_label">
         <property name="text">
          <string>Pic_Harkonnen</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="button_atreides">
         <property name="text">
          <string>Atreides</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="button_harkonnen">
         <property name="text">
          <string>Harkonnen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="button_ordos">
         <property name="text">
          <string>Ordos</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
